### ISSUE TITLE
Two requests running in the same core despite "2 server(s)" message

### ISSUE THREAD

**Issue by Athospd** [rexyai/RestRserve#161]
Hello, everyone. First of all, congrats for this package!

I need help: when I call my API simultaniously it operates in serial instead of parallel. I observe only one core been used during the operation even though '2 server(s)' shown in the console. I need guidance, please.

```
Rscript -e "relatorioSenna::iniciar_api_restrserver()"
{"timestamp":"2020-08-27 16:39:43.978917","level":"INFO","name":"Application","pid":13665,"msg":"","context":{"http_port":7676,"endpoints":{"HEAD":"/teste","GET":"/test"}}}
-- running Rserve in this R session (pid=13665), 2 server(s) --
```

Thank you!

**Comment by dselivanov** [comment]
That's not enough information to help you. Please report at least
sessioninfo as suggested in the issue template. If you are on windows,
don't expect it will serve requests in parallel.

**Comment by Athospd** [comment]
Hey, @dselivanov, I'm really sorry for the lack of info, here it is. But I think I found a potential interesting issues for you, regarding chrome browser. I made a gif, check it out: https://slack-files.com/TKA92V8UE-F019PGTP5M0-ce4b790858

Summary: 6 calls simultaniously for the same endpoint. 2 from curl, 2 from firefox and 2 from chrome. The only call that took longer to run was the second call from chrome!

Well, I'll close this issue once it has nothing to do with {RestRserve}.
Thank you very much for your time!!!
